vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -1782,6 +1782,24 @@ func TestAccResourceVSphereVirtualMachine_cloneWithBadSizeWithLinkedClone(t *tes
 	})
 }
 
+func TestAccResourceVSphereVirtualMachine_Instantclone(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			RunSweepers()
+			testAccPreCheck(t)
+			testAccResourceVSphereVirtualMachinePreInstantCloneCheck(t)
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccResourceVSphereVirtualMachineCheckExists(false),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccResourceVSphereVirtualMachineConfigInstant(),
```

Comment:
UndeclaredName

![image](https://user-images.githubusercontent.com/7771363/158680636-44d3e1a1-b65f-48e2-b8a5-44c1506dc6a8.png)

Reply:
So what change should I do regarding this?

Reply:
Can linked_clone and instant_clone be used at the same time? If not, there might been to be a ConflictsWith btween the two.